source/backend/dal/Repositories/DispositionFileRepository.cs

```diff
@@ -15,6 +16,11 @@ namespace Pims.Dal.Repositories
     /// </summary>
     public class DispositionFileRepository : BaseRepository<PimsDispositionFile>, IDispositionFileRepository
     {
+        private const string FILENUMBERPREFIX = "D";
```

Comment:
Please remove this. The D in front of the file number is for display only, we don't want the D in the database.

Reply:
Updated